src/rnanorm/__main__.py

```diff
@@ -44,6 +45,7 @@ def parse_args(args):
         "expression", type=str, help="tab-delimited file with gene expression data (genes in rows, samples in cols)",
     )
     parser.add_argument("--gene-lengths", type=str, help="tab-delimited file with gene lengths")
+    parser.add_argument("--gtf", type=str, help="Annotation file in GTF format")
```

Comment:
Should we make it more general? `--annotation`, `help="...in GTF/GFF3"...`

Reply:
Just done.